docs/source/alert_development_guide.rst

```diff
@@ -36,8 +36,7 @@ This test should pass and you will have confirmed you have a working environment
 
 At this point, begin development and periodically run your unit-tests locally with the following commands::
 
-  make build-tests
-  make run-tests TEST_CASE=tests/alerts/[YOUR ALERT TEST FILE].py
+  make build-tests && make run-tests TEST_CASE=tests/alerts/[YOUR ALERT TEST FILE].py
```

Comment:
We could use `make tests` instead, which calls builds-tests and run-tests.

Reply:
The syntax would be `make tests TEST_CASE=tests/alerts/[YOUR ALERT TEST FILE].py`?

Reply:
Yup!